index.js

```diff
@@ -126,13 +126,9 @@ cm.Item({
 });
 
 function checkLocation(opts) {
```

Comment:
I see, I think we should still find another way. We should be checking based on the embed element or link itself. That way when embeds are on different sites from vine or twitter (like reddit or 4chan or tumblr or someone's wordpress blog) we won't have to add specific handling.